### PR TITLE
Update Agama trigger command settings and extract as repo to openqa repo folder

### PR DESCRIPTION
**Pull request by tinawang123**
extract_as_repo for x86_64 aarch64 ppc64le

### CODE REVIEW

File: xml/obs/systemsmanagement:Agama:Devel.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     distri="opensuse"
     version="agama-9.0">
     <batch name="base" archs="x86_64 aarch64 ppc64le">
-        <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/" iso="1" media1="0"/>
+        <flavor name="agama-installer" folder="images/*/agama-installer:openSUSE/" iso="iso" extract_as_repo="1" media1="0"/>
```
Comment (rfan1) [inline xml/obs/systemsmanagement:Agama:Devel.xml:7]: I have one question here, is there any difference  for s390x?

Comment (tinawang123) [reply xml/obs/systemsmanagement:Agama:Devel.xml:7]: REPO_999=agama-installer.s390x-9.0.0-openSUSE-Build17.9.iso \
 s390x has ln_iso_to_repo, so it has this setting.

Comment (rfan1) [reply xml/obs/systemsmanagement:Agama:Devel.xml:7]: Thanks！

**Comment by foursixnine** [comment]
@rfan1 @tinawang123 you have commit rights?

**Comment by rfan1** [comment]
> @rfan1 @tinawang123 you have commit rights?

I don't think so

**Review by andrii-suse — APPROVED** [review]

**Comment by foursixnine** [comment]
> > @rfan1 @tinawang123 you have commit rights?
> 
> I don't think so

Yo both have now

**Pull request merged**